src/AccordionItemBody/index.js

```diff
@@ -8,7 +8,7 @@ export default class AccordionItemBody extends Component {
 
   getProps() {
     const props = {
-      [`aria-${this.props.expanded ? 'expanded' : 'hidden'}`]: true
+      [`hidden`]: !this.props.expanded
```

Comment:
See comments on AccordianItemTitle - according to W3C guidlines it should be the 'element that does the expanding' that has aria-expanded set.

Would be useful to hide the body if not expanded - that way, elements within the item body will not be 'tabbable' despite being invisible.

Reply:
Well spotted... thanks!

I've fixed this up; are you OK to merge?

Thanks,

Simon